app/uk/gov/hmrc/eeittadminfrontend/connectors/GformConnector.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.eeittadminfrontend.connectors
+
+
+import play.api.libs.json.JsValue
+import play.libs.Json
+import uk.gov.hmrc.eeittadminfrontend.WSHttp
+import uk.gov.hmrc.eeittadminfrontend.models.{FormTypeId, GformTemplate}
+import uk.gov.hmrc.play.config.ServicesConfig
+import uk.gov.hmrc.play.http.{HeaderCarrier, HttpGet, HttpPost, HttpResponse}
+import scala.concurrent.Future
+
+object GformConnector extends ServicesConfig {
+  val httpGet = WSHttp
+  val httpPost = WSHttp
+  val gformUrl = "http://localhost:9196/gform"
```

Comment:
How about suc way of making gformUrl? 
```
private val sc = new ServiceConfig{}
val gformUrl = "${sc.baseUrl("gform")}/gform"
```
(... and then you don't need to extend `ServiceConfig`, which can misleadingly suggest that `GformConnector` is kind of `ServiceConfig`, which is not true)